### ISSUE TITLE
"No such file or directory" when installing themes

### ISSUE THREAD

**Issue by Wrx95Pro** [ohitstom/spicetify-easyinstall#51]
Spicetify has had some path changes and I've yet to fully fix it, on holiday until tomorrow but I'll finish it and release it when I get back

_Originally posted by @ohitstom in https://github.com/ohitstom/spicetify-easyinstall/issues/32#issuecomment-1179687439_

Is it really fixed? Have the same issue
![image](https://user-images.githubusercontent.com/38694456/279192370-aaa54ce5-469e-4632-8512-50e5b44d7c4b.png)

**Comment by ohitstom** [comment]
if you notice the paths are not the same, this issue is usually caused by API timeout

the install completed successfully but you didn't fetch add-ons, you can go back to the main page then go to update then addons to install just that part again

**Comment by ohitstom** [comment]
however you are right in saying this issue was never fixed, I changed paths and it is still present but only for some

I've STILL yet to manage to recreate the issue so I cant fix it as of yet

**Comment by ohitstom** [comment]
> however you are right in saying this issue was never fixed, I changed paths and it is still present but only for some
> 
> I've STILL yet to manage to recreate the issue on my own hardware so I cant fix it as of yet

**Comment by panicthescaredycat** [comment]
Hello, Having the same issue here, I've gone back after it installs to ''update'' the addons, but i still get the same error..

**Comment by ohitstom** [comment]
> Hello, Having the same issue here, I've gone back after it installs to ''update'' the addons, but i still get the same error..

some people have had luck with using a vpn, however that may be more hassle than just installing spicetify the standard way. This installer was made in a time when spicetify was super unstable which isnt really the case anymore so its sorta become obsolete

**Comment by panicthescaredycat** [comment]
So... I got it to work.. weirdly by trying over and over again, closing and opening the program and retrying to "update add-ons" idk it was weird took a bit..

**Comment by ohitstom** [comment]
> So... I got it to work.. weirdly by trying over and over again, closing and opening the program and retrying to "update add-ons" idk it was weird took a bit..

yeah this error is super weird, I even consulted the people who make this package for python that seems to be causing the error and they couldn't figure it out. my thoughts are it has to be some networking problem with GitHub or something

**Comment by Bluemethyst** [comment]
im having the same error on first time install after resetting computer. any ideas?
